tests/modelgauge_tests/test_cli.py

```diff
@@ -66,6 +68,33 @@ def test_run_sut_demos(sut):
     assert re.search(r"Native response:", result.output)
 
 
+@patch("modelgauge.suts.demo_01_yes_no_sut.DemoYesNoSUT.translate_text_prompt")
+def test_run_sut_with_options(mock_translate_text_prompt):
```

Comment:
Good stuff. These CLI tests are kind of a pain to write, but I'm glad to see it.